io.github.JakubMelka.Pdf4qt.json

```diff
@@ -0,0 +1,53 @@
+{
+    "app-id": "io.github.JakubMelka.Pdf4qt",
+    "runtime": "org.kde.Sdk",
+    "runtime-version": "6.5",
+    "sdk": "org.kde.Sdk",
+    "command": "Pdf4QtViewerProfi",
+    "finish-args": [
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc"
+    ],
+    "cleanup": [
+        "/pdf4qt",
+        "/vcpkg"
+    ],
+    "modules": [ 
+        {
+            "name": "pdf4qt",
+            "buildsystem": "cmake-ninja",
+            "post-install": [
+                "echo 'Finishing BUILD - move plugins into right place'",
+                "./postbuild.sh"
+            ],
+            "config-opts": [
+                "-S /run/build/pdf4qt",
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DPDF4QT_INSTALL_QT_DEPENDENCIES=0",
+                "-DCMAKE_TOOLCHAIN_FILE=/run/build/pdf4qt/vcpkg/scripts/buildsystems/vcpkg.cmake",
+                "-DCMAKE_INSTALL_PREFIX=/app"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/JakubMelka/flathub/releases/download/v1/pdf4qt_vcpkg_loc.zip",
```

Comment:
this is not sources it seems
```
/run/build/pdf4qt/vcpkg/vcpkg: /run/build/pdf4qt/vcpkg/vcpkg: cannot execute binary file
```

This is the failure on aarch64.

Reply:
It is a C++ package manager, which runs in offline mode and builds dependencies. I have disabled aarch64.

Reply:
This is very disappointing to disable aarch64 for no real reason.

Reply:
Well, I plan to fix it, however, first I want to implement x86_64 first.